build-libssp.sh

```diff
@@ -37,36 +37,18 @@ export PATH="$PREFIX/bin:$PATH"
 : ${ARCHS:=${TOOLCHAIN_ARCHS-i686 x86_64 armv7 aarch64}}
 
 if [ ! -d libssp ]; then
-    svn checkout -q https://github.com/gcc-mirror/gcc/tags/releases/gcc-7.3.0/libssp
+    svn checkout -q svn://gcc.gnu.org/svn/gcc/tags/gcc_9_2_0_release/libssp 
+    patch -p1 < 0002-allow-building-outside-of-the-gcc-tree.patch
+    (cd libssp; autoreconf -fiv)
```

Comment:
I don't think autoreconf is one of the prerequisites already installed - can you check that the dockerfiles installs what's needed, and ideally also update the list of msys2 packages needed (listed in README.md)?

Reply:
OK
Also AFAIK on mac autoreconf is kinda different. Is it going to be a problem ?

Reply:
> Also AFAIK on mac autoreconf is kinda different. Is it going to be a problem ?

Hmm, well, right now we already require you to have CMake available somehow, to be able to build it. After this, we'd need you to provide both CMake and autotools. Not ideal but probably acceptable, as it's probably not built quite as much on macos as on linux and msys2. Long ago (10 years?) I remember that Xcode used to provide some version of autotools bundled, but a rather old version at the time. These days it's no longer bundled.

BTW, you need something like `COPY patches/libssp/*.patch ./patches/libssp/` in the dockerfiles, above/below the one that copies `build-libssp.sh`.

Reply:
Yes I tested in Docker and fixed the missing parts accordingly.
(and now I realize I need to update *ALL* Docker files)